core/services/keystore/master.go

```diff
@@ -209,18 +209,6 @@ func (ks *master) Migrate(vrfPssword string, f DefaultEVMChainIDFunc) error {
 		return err
 	}
 	if len(ethKeys) > 0 {
-		chainID, err := f()
```

Comment:
it seems like we should be doing this for all or none of the evm chains now that that there is no default, but i'm not sure which